org.eclipse.ui.genericeditor/src/org/eclipse/ui/internal/genericeditor/ReconcilerRegistry.java

```diff
@@ -65,71 +70,85 @@ public ReconcilerRegistry() {
 	 * Get the contributed {@link IReconciliers}s that are relevant to hook on
 	 * source viewer according to document content types.
 	 *
-	 * @param sourceViewer the source viewer we're hooking completion to.
-	 * @param editor       the text editor
-	 * @param contentTypes the content types of the document we're editing.
+	 * @param sourceViewer         the source viewer we're hooking completion to.
+	 * @param editor               the text editor
+	 * @param textViewerLifecycles the list of text viewer lifecycle to fill
+	 * @param contentTypes         the content types of the document we're editing.
 	 * @return the list of {@link IReconciler} contributed for at least one of the
 	 *         content types, sorted by most generic content type to most specific.
 	 */
 	public List<IReconciler> getReconcilers(ISourceViewer sourceViewer, ITextEditor editor,
-			Set<IContentType> contentTypes) {
+			List<IReconcilingStrategy> reconcilingStrategies, Set<IContentType> contentTypes) {
```

Comment:
I believe those changes are not so related to the plain addition of the interface and thus should be removed from the ocmmit.

Reply:
Removed